skbuild/src/harness.sk

```diff
@@ -1,5 +1,7 @@
 module Skbuild;
 
+const kBUFFER_LEN: Int = 1024;
```

Comment:
This seems pretty small to me. See e.g. https://git.savannah.gnu.org/cgit/coreutils.git/tree/src/ioblksize.h#n24 which reports on benchmarks done to decide to use 256KiB for buffers in GNU coreutils.